test/ExportVerilog/rtl-dialect.mlir

```diff
@@ -599,11 +599,10 @@ rtl.module @longvariadic(%a: i8) -> (%b: i8) {
 // CHECK-EMPTY:
 // CHECK-NEXT:   reg memory_r_en_pipe[0:0];
 // CHECK-EMPTY:
-// CHECK-NEXT:   wire _T = 1'h0;
 // CHECK-NEXT:   always_ff @(posedge clock)
-// CHECK-NEXT:     memory_r_en_pipe[_T] <= _T;
+// CHECK-NEXT:     memory_r_en_pipe[1'h0] <= 1'h0;
 // CHECK-NEXT:   initial
-// CHECK-NEXT:     memory_r_en_pipe[_T] = _T;
+// CHECK-NEXT:     memory_r_en_pipe[1'h0] = 1'h0;
```

Comment:
This change shouldn't happen, we should emit the 1 out of line to match the IR.  A future unrelated patch to RTLCleanup should handle this, since it is just a prettification

Reply:
Sure thing. Also the rationale for not inlining discussed in #895 makes sense. Will remove this.